Frontend/app/secureTest/secureTest.html

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+  <div class="col-xs-12">
+    <h1>
+      Test your Auth Token Here
```

Comment:
You didn't add instructions on how to see that you have the token though :2nd_place_medal: